Makefile

```diff
@@ -40,18 +40,26 @@ install: config.lua blocklist
 uninstall:
 	${RM} -r ${INSTALL_PATH}
 
-run: install ${EBPF_FILTERS_OBJS}
-	lunatik reload
-	xdp-loader load -m skb luaxdp0 ${EBPF_FILTERS_OBJS}
-	lunatik spawn dome/daemon
+start: ${EBPF_FILTERS_OBJS}
+	@sh dome.sh start
+	@sh dome.sh status
 
 stop:
-	xdp-loader unload --all luaxdp0
-	lunatik stop dome/daemon
+	@sh dome.sh stop
+	@sh dome.sh status
+
+restart:
+	@sh dome.sh restart
+	@sh dome.sh status
+
+status:
+	@sh dome.sh status
```

Comment:
perhaps we can remove commands from here and use officially the new script.. I put them here only for convenience.. a quick and dirty hack..